portal-api/src/v1/routes.js

```diff
@@ -28,25 +28,54 @@ const { ukefDecisionReport, unissuedFacilitiesReport } = require('./controllers/
 const dealImportBssEwcsController = require('./controllers/data-migration/deal-import-bss-ewcs.controller');
 const dealImportGefController = require('./controllers/data-migration/deal-import-gef.controller');
 
-const users = require('./users/routes');
 const { cleanXss, fileUpload } = require('./middleware');
+const checkApiKey = require('./middleware/headers/check-api-key');
 
+const users = require('./users/routes');
 const gef = require('./gef/routes');
 
-const authRouter = express.Router();
 const openRouter = express.Router();
-const authRouterAllowXss = express.Router();
 
+openRouter.route('/login').post(users.login);
+
+// 1. Request password reset
+openRouter.route('/users/reset-password').post(users.resetPassword);
+
+// 2. Password reset post request
+openRouter.route('/users/reset-password/:resetPwdToken').post(users.resetPasswordWithToken);
+
+// API Key Routes
+openRouter.route('/feedback').post(checkApiKey, feedback.create);
+openRouter.route('/user').post(checkApiKey, users.create); // This endpoint is only used by mock-data-loader, for setting up an initial user
+
+const authRouterAllowXss = express.Router();
```

Comment:
I think so, these routes come before `authRouter.use(cleanXss)` on line 74.

There were already some routes set up on this router before my changes. I'm not sure why they were set up to avoid the `cleanXss` check.

But I have moved the users routes to this one. Do you think they should they be moved to `authRouter` instead?

Reply:
`mock-data-loader` still works after moving the user endpoints to `authRouter` so I will move them there.